<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGrupoDialog</class>
 <widget class="QDialog" name="AddGrupoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyAcolito - Agrupar asistentes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="acolitoRC.qrc">
    <normaloff>:/acolitos/iconos/grupos</normaloff>:/acolitos/iconos/grupos</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="acolitoRC.qrc">:/acolitos/iconos/reserva</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> Los asistentes son separados de sus grupos y vueltos a colocar en uno nuevo, téngalo en cuenta.Si un grupo queda vacio, es eliminado y su habitación, de tenerla, queda libre.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Nombre</string>
        </property>
        <property name="icon">
         <iconset resource="acolitoRC.qrc">
          <normaloff>:/acolitos/iconos/buscar</normaloff>:/acolitos/iconos/buscar</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DNI</string>
        </property>
        <property name="icon">
         <iconset resource="acolitoRC.qrc">
          <normaloff>:/acolitos/iconos/buscar</normaloff>:/acolitos/iconos/buscar</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Email</string>
        </property>
        <property name="icon">
         <iconset resource="acolitoRC.qrc">
          <normaloff>:/acolitos/iconos/buscar</normaloff>:/acolitos/iconos/buscar</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Mostrar sólo los asistentes seleccionados</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="acolitoRC.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddGrupoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddGrupoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
